<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic Image Processing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="orig_frame">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>200</y>
      <width>400</width>
      <height>280</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="proc_frame">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>200</y>
      <width>400</width>
      <height>280</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSatus">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>140</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>140</y>
      <width>760</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="imgInv">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Inversion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imgHistEql">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram Equalization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imgGammaCor">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>60</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma Correction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imgReset">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>80</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGamma">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gamma_input">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrigImg">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>175</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcImg">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>175</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Processed Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open..</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
